### PR TITLE
Updated women_laws.html file.

### PR DESCRIPTION
**Pull request by AishwaryaKaranth**
## Related Issuse or bug
  - Improve Women_laws.html documentation.

Fixes: #117

I have formatted the existing documentation (women_laws.html) and added some CSS effects to each paragraph.

### CODE REVIEW

**Review by swapnilsparsh — COMMENTED** [review]
**Fix the errors**
e
ec

**Dont use italics, just highlight it with blue colour as you have done and in some laws there is no highlighted text so do some research and if you want to change the details then change it and highlight the required text.**
italics

**Comment by AishwaryaKaranth** [comment]
I'm not sure what is causing the error, I only edited women_laws.html file. As for the laws, I'll make the necessary changes.

**Comment by sakshikhachane** [comment]
> **Fix the errors**
> e
> ec

@swapnilsparsh I have checked this and found that the error was there before only.

> **Dont use italics, just highlight it with blue colour as you have done and in some laws there is no highlighted text so do some research and if you want to change the details then change it and highlight the required text.**
> italics

@AishwaryaKaranth Do the above changes.

File: main_app/templates/main_app/women_laws.html [hunk 1]
```diff
@@ -59,6 +59,15 @@
             margin-left: auto;
             margin-right: auto;
         }
+        .description{
+            margin-left: 0.5rem;
+            margin-right: 0.5rem;
+            border-left: 3px solid #a8dadc;
+            padding-left: 0.5rem;
```
Comment (sakshikhachane) [inline main_app/templates/main_app/women_laws.html:66]: Please add padding-right as well for this which is equal to padding-left. Also, increase the padding-left as well.

File: main_app/templates/main_app/women_laws.html [hunk 2]
```diff
@@ -97,16 +106,13 @@ <h1 class="wb-stl-heading1">
                     <h5>The Commission of Sati (prevention) Act(1987)</h5>
                 </div>
                 <div class="collapsible-body">
-                    <p>Sati (Prevention) Act, 1987 is a law enacted by Government of Rajasthan in 1987.
+                    <p class="description"><span style="background:#a8dadc">Sati (Prevention) Act</span>, 1987 is a law enacted by Government of Rajasthan in 1987.
```
Comment (sakshikhachane) [inline main_app/templates/main_app/women_laws.html:109]: @AishwaryaKaranth Please edit the content in bullet form for every law. Right now, it is less bit unreadable.

**Review by sakshikhachane — CHANGES_REQUESTED** [review]

**Comment by swapnilsparsh** [comment]
> > **Fix the errors**
> > e
> > ec
> 
> @swapnilsparsh I have checked this and found that the error was there before only.
> 
> > **Dont use italics, just highlight it with blue colour as you have done and in some laws there is no highlighted text so do some research and if you want to change the details then change it and highlight the required text.**
> > italics
> 
> @AishwaryaKaranth Do the above changes.

Run `python manage.py migrate` then this error will resolve.

**Comment by sakshikhachane** [comment]
@AishwaryaKaranth Any updates on this PR?

**Comment by AishwaryaKaranth** [comment]
> @AishwaryaKaranth Any updates on this PR?

Sorry for the delay, will submit a PR by tonight.

**Pull request closed without merging**